ide/editor.search/src/org/netbeans/modules/editor/search/SearchComboBoxEditor.java

```diff
@@ -176,14 +176,14 @@ public static void changeToOneLineEditorPane(JEditorPane editorPane) {
                     @Override
                     public void insertString(DocumentFilter.FilterBypass fb, int offset, String string, AttributeSet attr) throws BadLocationException {
                         if (string != null) {
-                            fb.insertString(offset, string.replaceAll("\\t", "").replaceAll("\\n", ""), attr); //NOI18N
```

Comment:
i don't think you have to escape `\n` or `\t` but it doesn't seem to hurt.
```java
        System.out.println("a\nb\nc".replaceAll("\\n", ""));
        System.out.println("a\nb\nc".replaceAll("\n", ""));
        System.out.println("a\nb\nc".replace("\n", ""));
```
give equal results